backend/internal/service/webauthn_service.go

```diff
@@ -120,17 +120,25 @@ func (s *WebAuthnService) VerifyRegistration(ctx context.Context, sessionID, use
 		tx.Rollback()
 	}()
 
-	// Load & delete the session row
+	// Load the session row first (MySQL doesn't support RETURNING in DELETE)
 	var storedSession model.WebauthnSession
 	err := tx.
 		WithContext(ctx).
-		Clauses(clause.Returning{}).
-		Delete(&storedSession, "id = ?", sessionID).
+		First(&storedSession, "id = ?", sessionID).
 		Error
 	if err != nil {
 		return model.WebauthnCredential{}, fmt.Errorf("failed to load WebAuthn session: %w", err)
 	}
 
+	// Delete the session after loading
+	err = tx.
```

Comment:
Can you do this in 2 queries just for MySQL? PG and SQLite support this as a single statement, so there's one less roundtrip